File: src/main/resources/output/IFC4X3_js/Entities/IfcStructuralConnectionCondition/IfcFailureConnectionCondition/IfcFailureConnectionCondition.js
class IfcFailureConnectionCondition extends IfcStructuralConnectionCondition {
    constructor() {
        /** @type {IFCFORCEMEASURE} */
        this.TensionFailureX = null;
        /** @type {IFCFORCEMEASURE} */
        this.TensionFailureY = null;
        /** @type {IFCFORCEMEASURE} */
        this.TensionFailureZ = null;
        /** @type {IFCFORCEMEASURE} */
        this.CompressionFailureX = null;
        /** @type {IFCFORCEMEASURE} */
        this.CompressionFailureY = null;
        /** @type {IFCFORCEMEASURE} */
        this.CompressionFailureZ = null;
    }
}

File: src/main/resources/output/IFC4X3_csharp/Entities/IfcStructuralConnectionCondition/IfcFailureConnectionCondition/IfcFailureConnectionCondition.cs
public class IfcFailureConnectionCondition : IfcStructuralConnectionCondition
{
    public IfcForceMeasure TensionFailureX { get; set; }
    public IfcForceMeasure TensionFailureY { get; set; }
    public IfcForceMeasure TensionFailureZ { get; set; }
    public IfcForceMeasure CompressionFailureX { get; set; }
    public IfcForceMeasure CompressionFailureY { get; set; }
    public IfcForceMeasure CompressionFailureZ { get; set; }
}
